docker-traceability-plugin/src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction.java

```diff
@@ -564,18 +567,69 @@ private synchronized void load() {
             LOGGER.log(Level.SEVERE, "Failed to load the configuration from config path = "+config,e);
         }
     }
-    
+
+    /**
+     * Represents the different response formats (JSON, pretty JSON, XML).
+     */
+    private enum ResponseFormat {
+        JSON("application/json;charset=UTF-8", false),
+        PRETTYJSON("application/json;charset=UTF-8", true),
+        XML("text/xml;charset=UTF-8", false);
```

Comment:
@jtnord XML and XMLPRETTY will be supported when this ticket is done https://issues.jenkins-ci.org/browse/JENKINS-28727